<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgNewOp</class>
 <widget class="QWidget" name="DlgNewOp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Operation</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Label:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDateEdit" name="m_pEdtDate"/>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="m_chkbxReport">
     <property name="text">
      <string>Carry forward</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="m_pEdtLabel"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Date:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Item</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Value:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Comment:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="m_pEdtComment"/>
   </item>
   <item row="8" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>11</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdd">
       <property name="statusTip">
        <string>add a new operation</string>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="m_pCbxItem"/>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="m_EdtValue"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
